### DISCUSSION TITLE
Extending resolvers

### DISCUSSION THREAD

**Discussion by merko30** [MichalLytek/typegraphql-prisma#232] [Q&A]
I'm not sure if this is possible. I would like to "extend" generated resolvers so each "create*" mutation takes the user id from the request and connect a record I'm creating with the user.

Without that, I need to create a custom mutation for each "create*" mutation.

**Comment by MichalLytek** [comment]
I think that's not easily possible. There's no generic single crud resolver you could overwrite the implementation.

You can try to use TypeGraphQL middlewares to inject some custom logic.
